analyze_map.go

```diff
@@ -45,6 +45,10 @@ var mapBodies = map[string]mapParameterMasks{
 	"otel_exporter": {
 		defaultMasks: ngxConfTake1,
 	},
+	"oidc_provider": {
+		specialParameterMasks: map[string]uint{"session_timeout": ngxConf1More, "scope": ngxConf1More},
+		defaultMasks: ngxConfTake1,
+	},
```

Comment:
Can you explain why [`oidc_provider`](https://nginx.org/en/docs/http/ngx_http_oidc_module.html#oidc_provider) is treated as [`map`](https://nginx.org/en/docs/http/ngx_http_map_module.html#map)-like directive? The syntax and behavior feel a bit different. To me it seems `oidc_provider` has more in common with block directives that take arguments, e.g. `location`.